src/types.ts

```diff
@@ -38,13 +54,25 @@ export enum Language {
 
 export interface GameStartType {
   slpVersion: string | null;
+  timerType: TimerType | null;
+  gameMode: GameModeType | null;
+  friendlyFireEnabled: boolean | null;
+  bombRainEnabled: boolean | null;
   isTeams: boolean | null;
-  isPAL: boolean | null;
+  itemSpawnBehavior: ItemSpawnBehaviorType | null;
+  selfDestructScoreValue: number | null;
   stageId: number | null;
+  startingTimerSeconds: number | null;
+  enabledItems: number | null;
+  damageRatio: number | null;
```

Comment:
Why are there so many duplicate fields? Whatever fields exist in `GameInfoType` can we remove from `GameStartType`?